matchingengine/OrderBook.go

```diff
@@ -135,7 +135,24 @@ func (ob *orderBook) AddAskOrder(ask *models.Ask) {
 	// in case of stoploss order, add it to stoploss queue and return
 	if ask.OrderType == models.StopLoss {
 		l.Debugf("Adding stopLoss with ask_id %d to the queue", ask.Id)
-		ob.askStoploss.Push(ask)
+		stockCopy, _ := models.GetStockCopy(ask.StockId)
+
+		if stockCopy.CurrentPrice <= ask.Price {
+
+			db := utils.GetDB()
```

Comment:
Can we just call `ob.triggerStopLosses` here instead of effectively rewriting the code? Any reason why that won't work?